lib/coophub_web/templates/layout/app.html.eex

```diff
@@ -19,6 +19,12 @@
     <meta name="msapplication-config" content="<%= Routes.static_path(@conn, "/icons/browserconfig.xml") %>">
     <meta name="theme-color" content="#ffffff">
 
+    <meta property="og:title" content="CoopHub · Open source from co-operatives" />
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://coophub.io" />
+    <meta property="og:description" content="Repos and projects open source from co-operatives around the world" />
```

Comment:
I would change it to: Open source projects and repositories from co-operatives around the world!